man/systemd-cryptsetup-generator.xml

```diff
@@ -144,6 +144,19 @@
         to the one specified by <varname>rd.luks.key=</varname> or
         <varname>luks.key=</varname> of the corresponding UUID, or the
         password file that was specified without a UUID.</para>
+
+        <para>It is also possible to specify an external device which
+        should be mounted before we attempt to unlock the LUKS device.
+        systemd-cryptsetup will use password file stored on that
+        device.
```

Comment:
The syntax should be explained, e.g. by adding "by appending a colon and a device identifier".